sensors/ev3_uart_sensor_defs.c

```diff
@@ -113,13 +155,19 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 				 *    sensor does not respond to the keep-alive
 				 *    sent from the EV3 brick. As a result, the
 				 *    sensor will time out and reset.
+				 *    However, writing a special factory string
+				 *    to the sensor directly while in this mode
+				 *    may trigger hardware recalibration.
+				 *    Proceed only at your own risk though, as
+				 *    this will likely erase the
+				 *    factory-provided calibration.
 				 *
 				 * @name_footnote: [#lego-ev3-color-mode5]_
 				 * @description: Calibration ??? - sets LED color to red, flashing every 4 seconds, then goes continuous
-				 * @value0: ???
-				 * @value1: ???
-				 * @value2: ???
-				 * @value3: ???
+				 * @value0: ??? (0 to 1023???)
```

Comment:
According to my notes, this mode has 65535 reported as the max value (both raw and SI units).

Reply:
Yes, that is consistent with LEGO's `typedata.rcf`, I was guessing that this is coming from ADC, so >1023 wouldn't make sense. I will remove the change.

Reply:
On the stock firmware the values indeed exceed 1023, so the 65535 upper bound is likely to be true.

Reply:
Added 65535 as the upper bound :+1: